2-Pricing/app.css

```diff
@@ -0,0 +1,355 @@
+    padding: 0 0.5rem;
+    text-decoration: none;
+    color: white;
+    transition: all 0.35s ease-in-out;
+}
+
+h2 { margin: 0;}
+
+.selected-view {
+    color: black;
+    background-color: white;
+    font-weight: bold;
+}
+
+.view-left {
+    border-radius: 4rem 0 0 4rem;
+}
+
+.view-right {
+    border-radius: 0 4rem 4rem 0;
+}
+
+
+.container, .heading-section, .subscriptions,
+.card, .card-top, .card-bottom, .oval {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
+.container {
+    padding: 3.75rem;
+    gap: 3.125rem;
+}
+
+.heading-section {
+    max-width: 21.875rem;
+    line-height: 1.5rem;
+    gap: 0.9375rem;
+}
+
+.subscriptions {
+    flex-wrap: wrap;
+    gap: var(--spacing);
+}
+
+h1, h4 {
+    margin: 0;
+    text-align: center;
+}
+
+h1 {
+    font-family: 'Ubuntu', sans-serif;
+    font-weight: bolder;
+    font-size: 1.5rem;
+}
+
+h4 {
+    font-weight: bold;
+    font-size: 1rem;
+}
+
+p {
+    transition: 0.2s ease-in-out;
+    font-size: 0.8125;
+}
+
+.card {
+    box-sizing: border-box;
+    padding: 1.875rem 0.315rem;
+    width: 14.6875rem;
+    height: 25rem;
+    border: solid 1px white;
+    border-radius: 1%;
+    transition: 0.2s ease-in-out;
+}
+
+.card:hover {
+    height: 27rem;
+    width: 16.25rem;
+    margin: -1rem -0.785rem;
+}
+
+.card:hover .card-price { font-size: 2rem; }
+.card:hover .card-title { font-size: 1.5rem; }
+.card:hover p { font-size: 1rem; }
+.card:hover .button { font-size: 1rem; }
+
+.card-top {
+    justify-content: space-between;
+    height: 70%;
+    width: 100%;
+    gap: 0.685rem;
+    border-bottom: solid 1px gray;
+}
+
+.card-title {
+    transition: 0.2s ease-in-out;
+    opacity: 0.7;
+    font-size: 1.125rem;
+    font-weight: bolder;
+}
+
+.oval {
+    justify-content: center;
+    width: 5rem;
+    height: 5rem;
+    border-radius: 100%;
+    background-color: white;
+}
+
+.bravo-shadow { filter: drop-shadow(0 0.75rem 1.25rem var(--bravo-color)); }
+.charlie-shadow{ filter: drop-shadow(0 0.75rem 1.25rem var(--charlie-color)); }
+.extra-shadow { filter: drop-shadow(0 0.75rem 1.25rem var(--extra-color)); }
+.featured-shadow { filter: drop-shadow(0 0.75rem 1.25rem var(--featured-color)); }
+
+.bravo { color: var(--bravo-color) }
+.charlie { color: var(--charlie-color) }
+.extra { color: var(--extra-color) }
+
+
+.card-price {
+    transition: 0.2s ease-in-out;
+    font-size: 1.5rem;
+}
+
+.card-bottom {
+    height: 30%;
+    width: 100%;
+    justify-content: space-between;
+}
+
+.primary-btn {
+    font-size: 0.8125;
+    text-align: center;
+    border: none;
+    width: 50%;
+    padding: 0.625rem;
+    border-radius: 1.5rem;
+    background-color: white;
+    transition: 0.2s ease-in-out;
+    cursor: pointer;
+}
+
+.featured {
+    position: relative;
+    overflow: hidden;
+    background-color: white;
+    opacity: 1;
+    color: gray;
+    font-weight: bolder;
+    border: unset;
```

Comment:
What is this for?

Reply:
Those are the custom styles for the `featured` card. But well spotted. that `opacity: 1;` here is redundant. (The border is unset so that the orange "tag" at the top left doesn't have a white border on the edges of the card)

Reply:
Great, my comment wasn't very specific, but I meant the border haha, I thought it would only highlight that line. Thanks for the explanation!